alchemist-loading/src/main/kotlin/it/unibo/alchemist/loader/providers/YamlProvider.kt

```diff
@@ -20,21 +21,28 @@ import java.net.URL
  */
 object YamlProvider : AlchemistModelProvider {
 
+    private val loaderOptions: LoaderOptions = LoaderOptions().withMaxAliasesForCollections()
```

Comment:
Uhmm... why is this not in the `companion object`, assuming the implementation is thread-safe?

Reply:
ah, because it's an `object` already